src/server.rs

```diff
@@ -0,0 +1,61 @@
+use crate::verkle::{get_rlp, decode_block, verification};
+use crate::dot::to_dot;
+use actix_web::{web::{self, Json}, App, HttpServer};
+use crate::types::{VerkleReq, VerkleResp};
+use crate::Config;
+use tokio::process::Command;
+
+async fn get_block_info(data: Json<VerkleReq>) -> Result<Json<VerkleResp>, crate::error::Error> {
+    let block_number = data.into_inner().block_number;
+    let block_rlp = get_rlp(block_number).await?;
+    let block = decode_block(block_rlp.clone())?;
+
+    if block_number < 2 {
+        return Ok(Json(VerkleResp { image: "incorrect block number".to_owned() }));
+    }
```

Comment:
Why is `block_number == 1` invalid when we do only `block_number - 1` further?

Reply:
I don't really know what we have to put in `parent_root` with `block_number == 1`

Reply:
Wouldn't it work if it would be a genesis block with number 0?